framework/src/main/java/org/checkerframework/common/value/util/Range.java

```diff
@@ -151,29 +163,40 @@ public int hashCode() {
         return Objects.hash(from, to);
     }
 
+    /**
+     * Compare two ranges in a type safe manner for equality without incurring the cost of an
+     * instanceof check such as equals(Object) does.
```

Comment:
How about using this method from `equals(Object)`, to make sure the logic is consistent and there is no duplication?

Reply:
Yes. That change is in my local working directory.  
I am having a hell of a time getting gradlew reformat or simply format to work so my pace is slower than I wanted.

Reply:
Go ahead and push.  I can reformat for you.

Reply:
Your problems suggest that the formatter scripts don't run well under Windows.  Sorry about that.

I don't know whether the problem is in the Checker Framework's `build.gradle` file or in the [run-google-java-format](https://github.com/plume-lib/run-google-java-format) project itself.  If you could let us know the symptoms, perhaps we could help.

Another alternative is to install Windows Subsystem for Linux and run the reformat command there.  I know that works (though it's a rather heavyweight solution and doesn't help other Windows users).

Reply:
I committed the change. If it fails for format, please reformat.